File: src/main/java/eterea/report/factura/domain/model/CodigoQR.java
package eterea.report.factura.domain.model;

import lombok.*;

import java.math.BigDecimal;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class CodigoQR {

    private int ver = 1;
    private String fecha = "";
    private String cuit = "";
    private int ptoVta = 0;
    private int tipoCmp = 0;
    private long nroCmp = 0;
    private BigDecimal importe = BigDecimal.ZERO;
    private String moneda = "";
    private int ctz = 1;
    private int tipoDocRec = 0;
    private BigDecimal nroDocRec = BigDecimal.ZERO;
    private String tipoCodAut = "E";
    private String codAut = "";

}

File: src/main/java/eterea/report/api/rest/kotlin/model/mapper/CodigoQR.kt
package eterea.report.api.rest.kotlin.model.mapper

import java.math.BigDecimal

data class CodigoQR(

    var ver: Int = 1,
    var fecha: String = "",
    var cuit: String = "",
    var ptoVta: Int = 0,
    var tipoCmp: Int = 0,
    var nroCmp: Long = 0,
    var importe: BigDecimal = BigDecimal.ZERO,
    var moneda: String = "",
    var ctz: Int = 1,
    var tipoDocRec: Int = 0,
    var nroDocRec: BigDecimal = BigDecimal.ZERO,
    var tipoCodAut: String = "E",
    var codAut: String = ""

)
